pt-BR/action_controller_overview.md

```diff
@@ -4,19 +4,18 @@
 Action Controller Overview
 ==========================
 
-In this guide you will learn how controllers work and how they fit into the request cycle in your application.
-
-After reading this guide, you will know:
-
-* How to follow the flow of a request through a controller.
-* How to restrict parameters passed to your controller.
-* How and why to store data in the session or cookies.
-* How to work with filters to execute code during request processing.
-* How to use Action Controller's built-in HTTP authentication.
-* How to stream data directly to the user's browser.
-* How to filter sensitive parameters so they do not appear in the application's log.
-* How to deal with exceptions that may be raised during request processing.
-
+Nesse guia você irá aprender como `controllers` trabalham e como eles se encaixam no ciclo de requestes da sua aplicação.
+
+Depois de ler este guia, você irá saber:
+
+* Como seguir o fluxo de um requeste através de um controller.
```

Comment:
```suggestion
* Como seguir o fluxo de uma requisição através de um controller.
```

Reply:
Acho que o controller também pode ir como itálico